.travis.yml

```diff
@@ -25,15 +25,14 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 5.5
-      env:
-        - EXECUTE_CS_CHECK=true
     - php: 5.6
       env:
+        - EXECUTE_CS_CHECK=true
```

Comment:
Let's move this into the PHP 7 target, so that we can get results earlier, please. (That's what we've done in other components as well.)